docs/src/main/asciidoc/cli-tooling.adoc

```diff
@@ -332,4 +336,24 @@ Note: Output will vary depending on the build tool your project is using (`maven
 .Compatibility with Quarkus 1.x
 ====
 The version 2 Quarkus CLI can not be used with 1.x Quarkus projects or releases. Use the maven/gradle plugins when working with Quarkus 1.x projects.
-====
\ No newline at end of file
+====
+
+== Shell autocomplete and aliases
+
+Automatic command completion is available for Bash and Zsh:
+
+[source,shell]
+----
+# Setup autocompletion in the current shell
+source <(quarkus completion)
```

Comment:
This looks weird to me? Just checking it's how it should be.

Reply:
https://kubernetes.io/docs/tasks/tools/included/optional-kubectl-configs-bash-linux/#enable-kubectl-autocompletion